src/api-spec/protobuf/buf.yaml

```diff
@@ -1,10 +1,7 @@
 version: v1
-deps:
-  - buf.build/tdex-network/tdex-protobuf
-  - buf.build/tdex-network/tdex-daemon
 breaking:
```

Comment:
why we do not need the deps anymore

Reply:
We build stubs with `buf generate buf.build/tdex-network/tdex-protobuf && buf generate buf.build/tdex-network/tdex-daemon`. `buf.build/tdex-network/tdex-protobuf` and `buf.build/tdex-network/tdex-daemon` are not external dependencies like `buf.build/googleapis/googleapis` could be. So no need to list them in `deps`. Isn't it?

Reply:
I think if you list them, with buf generate, it will automatically execute avoinding doing `buf generate buf.build/tdex-network/tdex-protobuf && buf generate buf.build/tdex-network/tdex-daemon`

Reply:
I just get `Failure: no .proto target files found`

Reply:
werod, lets keep like this